kubernetes/linux/setup.sh

```diff
@@ -9,37 +9,13 @@ sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     dpkg-reconfigure --frontend=noninteractive locales && \
     update-locale LANG=en_US.UTF-8
 
-wget https://github.com/Microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_v1.10.0-1/omsagent-1.10.0-1.universal.x64.sh
+#install oneagent - Official bits (05/17/2021)
+wget https://github.com/microsoft/Docker-Provider/releases/download/05172021-oneagent/azure-mdsd_1.10.0-build.master.212_x86_64.deb
 
-#create file to disable omi service startup script
-touch /etc/.omi_disable_service_control
-
-chmod 775 $TMPDIR/*.sh
-
-#Extract omsbundle
-$TMPDIR/omsagent-*.universal.x64.sh --extract
-mv $TMPDIR/omsbundle* $TMPDIR/omsbundle
-#Install omi
-/usr/bin/dpkg -i $TMPDIR/omsbundle/110/omi*.deb
-
-#Install scx
-/usr/bin/dpkg -i $TMPDIR/omsbundle/110/scx*.deb
-#$TMPDIR/omsbundle/bundles/scx-1.6.*-*.universal.x64.sh --install
-
-#Install omsagent
-
-/usr/bin/dpkg -i $TMPDIR/omsbundle/110/omsagent*.deb
-#/usr/bin/dpkg -i $TMPDIR/omsbundle/100/omsconfig*.deb
-
-#install oneagent - Official bits (05/2021)
-wget https://github.com/microsoft/Docker-Provider/releases/download/05112021-oneagent/azure-mdsd_1.8.0-build.master.189_x86_64.deb
 /usr/bin/dpkg -i $TMPDIR/azure-mdsd*.deb
 cp -f $TMPDIR/mdsd.xml /etc/mdsd.d
 cp -f $TMPDIR/envmdsd /etc/mdsd.d
 
-#Assign permissions to omsagent user to access docker.sock
-sudo apt-get install acl
```

Comment:
did we check on both docker & containerd nodes before removing ?

Reply:
Yes, this being validated. we only use docker.sock for the telemetry for the docker version and which is already validated.